modules/tensor_mechanics/src/materials/ComputeMultipleInelasticStress.C

```diff
@@ -92,7 +95,9 @@ ComputeMultipleInelasticStress::ComputeMultipleInelasticStress(const InputParame
     _consistent_tangent_operator(_num_models),
     _cycle_models(getParam<bool>("cycle_models")),
     _matl_timestep_limit(declareProperty<Real>("matl_timestep_limit")),
-    _identity_symmetric_four(RankFourTensor::initIdentitySymmetricFour)
+    _identity_symmetric_four(RankFourTensor::initIdentitySymmetricFour),
+    _include_damage(isParamValid("damage_model") ? true : false),
+    _damage_model(nullptr)
```

Comment:
```suggestion
    _damage_model(isParamValid("damage_model") ? getMaterial("damage_model") : nullptr)
```

Reply:
We want to get the damage model as `DamageBase * _damage_model` and not as a material property. Not sure if this will work.

Reply:
`getMaterial` is to `getMaterialByName` as `getMaterialProperty` is to `getMaterialPropertyByName`.

Reply:
I see what you mean. However, this gives a pointer to the `Material` class and we would need another member to convert this to `DamageBase`.

Reply:
Right. Just do dynamic cast in the initializer list and then change the check below to check 

```
if (isParamValid("damage_model") && !_damage_model)
  paramError(...)
```